### ISSUE TITLE
Updated mobile menu flyout

### ISSUE THREAD

**Issue by cmajel** [GSA/digitalgov-pra#170]
from 5/6 styling sync.

As time allows, I'd recommend that we adjust the styling of our mobile menu flyout.

**Current**
<img width="407" alt="Screen Shot 2019-05-08 at 3 12 32 PM" src="https://user-images.githubusercontent.com/2374206/57401492-c3cb4f80-71a3-11e9-846e-c8fafadc1739.png">

**Proposed - expanding**
![mobile flyout --expanded](https://user-images.githubusercontent.com/2374206/57402482-6c2de380-71a5-11e9-8f47-093a5d253335.jpg)

This is desirable for a few reasons:
* The 100% width makes it easier to scan and read the list (as opposed to a 2/3 width flyout)
* It keeps the menu / close button in the same space, so a user can easily toggle the menu without changing position

If this is cumbersome to implement, here is a cleaned up flyout menu as an option

![Mobile flyout -- panel](https://user-images.githubusercontent.com/2374206/57402695-da72a600-71a5-11e9-862d-83886363bdc8.jpg)

A few notes on the changes:
* Labels the Menu flyout directly
* Cleans up spacing of items so they are easier to scan.
* Creates bigger close area for easy toggling

**Comment by hursey013** [comment]
I agree with all of these changes, but I actually think it may be best to propose these updates upstream to USWDS because I think the improvements are valuable for all of the reasons you mentioned and could benefit others as well.
